src/cms/preview-templates/HomePreview.tsx

```diff
@@ -1,13 +1,17 @@
 import React, { FunctionComponent } from 'react'
 import { PreviewProps, getJSON } from './helpers'
-import { HomePageComponent } from '../../components/Home'
-import { HomeData, HomeDataOnDisk } from '../../data/home'
+import { HomePageComponent } from '../../components/Home/component'
+import { HomeData, HomeDataOnDisk } from '../../data/home/types'
 import { createPreview } from './Preview'
 import { translate, Locale } from '../i18n'
 import { filterByLocale } from '../../data/i18n'
+import { ProjectData } from '../../data/project/types'
 import * as R from 'ramda'
 
-const Preview = createPreview<HomeDataOnDisk, HomeData>()
+const Preview = createPreview<
+  HomeDataOnDisk,
+  HomeData & { projects: ProjectData[] }
```

Comment:
Why not
```suggestion
  HomeData
```

isn't the project data part of the home data?